internal/controller/pathwaysjob_controller.go

```diff
@@ -471,19 +471,34 @@ func MakeSuccessPolicy(pw *pathwaysjob.PathwaysJob) *jobsetv1alpha2.SuccessPolic
 	}
 }
 
+// Pick the RM Image based on whether custom image or Pathways version are provided.
+func MakeResourceManagerAndWorkerImage(pw *pathwaysjob.PathwaysJob) string {
+	if pw.Spec.CustomComponents != nil && pw.Spec.CustomComponents.PathwaysServerImage != "" {
+		return pw.Spec.CustomComponents.PathwaysServerImage
+	} else {
+		return fmt.Sprintf("us-docker.pkg.dev/cloud-tpu-v2-images/pathways/server:%s", makeImageTagUsingPathwaysVersion(pw))
+	}
+}
+
 // Constructs the Pathways resource manager container spec for the underlying JobSet
 func MakeResourceManagerContainer(pw *pathwaysjob.PathwaysJob, isInitContainer bool) (*corev1.Container, error) {
+	args := []string{
+		"--server_port=29001",
```

Comment:
Maybe refactor to a constant?

Reply:
Not needed since this is the only place.